templates/includes/sidebar.html

```diff
@@ -0,0 +1,9 @@
+<div class="sidebar">
+    <ul>
+        <li><a href="#">Home page</a></li>
+        <li><a href="#">Manufacturers</a></li>
+        <li><a href="#">Cars</a></li>
+        <li><a href="#">Drivers</a></li>
+    </ul>
+</div>
+
```

Comment:
It's a good practice to end HTML files with a blank line for better formatting and to avoid potential issues with some version control systems.